native/acter/src/api/verification.rs

```diff
@@ -516,21 +505,23 @@ async fn request_verification_handler(
     txn_id: Option<OwnedTransactionId>,
     sender: OwnedUserId,
     methods: Option<Vec<VerificationMethod>>,
-) {
+) -> Result<()> {
     info!(
         "Accepting verification request from {}",
         request.other_user_id()
     );
     if let Some(methods) = methods {
-        request
-            .accept_with_methods(methods)
-            .await
-            .expect("Unable to accept verification request with methods");
+        let result = request.accept_with_methods(methods).await;
+        if let Err(e) = result {
+            error!("Unable to accept verification request with methods");
+            return Err(e.into());
+        }
```

Comment:
why can't we just do ...
```suggestion
        let result = request.accept_with_methods(methods).await?;
```